src/Views/Admin/AddTeamComponents/ManageMembers.jsx

```diff
@@ -55,33 +73,19 @@ const ManageMembers = () => {
     }
   };
 
-  const membersArray = [
-    {
-      memberName: "TAKERU SATOH",
-      memberProfileType: "PLAYER",
-      memberAddress: "TOKYO, JAPAN",
-      discordLink: "https://discord.com",
-      fbLink: "https://fb.com",
-      twitchLink: "https://twitch.com",
-    },
-  ];
-
   const submitData = async () => {
-    if (IGN && profileFile && name && address) {
+    if (
+      memberData.IGN &&
+      profileFile &&
+      memberData.name &&
+      memberData.address
```

Comment:
For this part Ty can we make it in 1 condition lang then if it's really needed can we create a validation sa frontend nga input nga required? 

Like:

 if(memberData){
/* your usual code below*/